docs/releases.rst

```diff
@@ -4,6 +4,16 @@
     Releases
 ======================
 
+tmt-1.56.0
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The :ref:`tmt try</stories/cli/try>` command's interactive session
+now includes a new ``-h`` option to run shell commands directly
+on the host (not on provisioned guests). Users can exit cleanly
+with ``'q'``, KeyboardInterrupt, or EOFError, and failed commands
```

Comment:
Users don't really deal with `EOFError`s or `KeyboardInterrupt`s. Instead, they deal with "pressing `Ctrl-D`" or similar actions.

Also, I would propose using different command rather than simple `q`, users may easily have a script called `q`. `\q` is pretty common, backslash making it more special and less prone to conflicts with actual commands in the live Linux system.